cli/commands/mutator/info.go

```diff
@@ -0,0 +1,81 @@
+package mutator
+
+import (
+	"errors"
+	"io"
+	"strconv"
+
+	"github.com/sensu/sensu-go/cli"
+	"github.com/sensu/sensu-go/cli/commands/helpers"
+	"github.com/sensu/sensu-go/cli/elements/list"
+	"github.com/sensu/sensu-go/types"
+	"github.com/spf13/cobra"
+)
+
+// InfoCommand defines the 'mutator info' subcommand
+func InfoCommand(cli *cli.SensuCli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "info [NAME]",
+		Short:        "show detailed mutator information",
+		SilenceUsage: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			format, _ := cmd.Flags().GetString("format")
+
+			if len(args) != 1 {
+				_ = cmd.Help()
+				return errors.New("invalid argument(s) received")
+			}
+
+			// Fetch the mutator from API
+			name := args[0]
+			r, err := cli.Client.FetchMutator(name)
+			if err != nil {
+				return err
+			}
+
+			if format == "json" {
+				if err := helpers.PrintJSON(r, cmd.OutOrStdout()); err != nil {
+					return err
+				}
+			} else {
+				printToList(r, cmd.OutOrStdout())
+			}
+
+			return nil
```

Comment:
Maybe we could simplify to

```golang
if format == "json" {
  return helpers.PrintJSON(r, cmd.OutOrStdout())
}
printToList(r, cmd.OutOrStdout())
return nil
```

Reply:
(On a related note.. honestly `printToList` should probably return an error, but, that's an issue with my `list` package and totally out of scope.)

Reply:
oh yeah, that is much cleaner. Thanks for the suggestion, I'll change it!

Reply:
Should I add an issue for printToList returning an error?

Reply:
No worries, I can do it.